docker-compose.yml

```diff
@@ -538,6 +589,29 @@ services:
       - "${REDIS_PORT}"
     logging: *logging
 
+  broker:
```

Comment:
This should be called `kafka`. I understand an argument can be made this is the Kafka broker, it is well known as `kafka`.  This will also require changes in the .env file for the Kafka service address variable.